exporter/containerimage/writer.go

```diff
@@ -369,6 +547,22 @@ func emptyImageConfig() ([]byte, error) {
 	return dt, errors.Wrap(err, "failed to create empty image config")
 }
 
+func emptyImageConfig() ([]byte, error) {
```

Comment:
could we add some code sharing between this and `defaultImageConfig` function.

Reply:
Have simplified them both, since now we shouldn't need to have the extra `Variant` property defined, since OCI defines it.